ci/build_python.sh

```diff
@@ -13,20 +13,31 @@ rapids-logger "Begin py build"
 
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 
+version=$(rapids-generate-version)
+git_commit=$(git rev-parse HEAD)
+export RAPIDS_PACKAGE_VERSION=${version} 
+echo "${version}" | tr -d '"' > VERSION
+
 # TODO: Remove `--no-test` flags once importing on a CPU
 # node works correctly
+version_file_pylibraft="python/pylibraft/pylibraft/_version.py"
```

Comment:
for succinctness and consistency with the other PRs below, can we put all of these `_version.py` file updates in a loop?

- https://github.com/rapidsai/cuspatial/pull/1285/files#diff-8d95070af6922d82099b7c3430ea734e6138e2f8e577632c812bc76b298f608bR18-R21
- https://github.com/rapidsai/cudf/pull/14285/files#diff-8d95070af6922d82099b7c3430ea734e6138e2f8e577632c812bc76b298f608bR17-R19